crates/bevy_window/src/window.rs

```diff
@@ -602,12 +602,24 @@ impl Window {
     }
 }
 
+/// Describes the information needed for a window creation.
+///
+/// This should be set up before adding the [`WindowPlugin`](crate::WindowPlugin). Most of these settings can also later
+/// be configured through the [`Window`](crate::Window) resource.
+///
+/// See [`examples/window/window_settings.rs`](https://github.com/bevyengine/bevy/blob/latest/examples/window/window_settings.rs) for usage.
 #[derive(Debug, Clone)]
 pub struct WindowDescriptor {
+    /// The requested logical width of the window's client area.
+    /// May vary from the physical width due to different pixel density on different monitors.
     pub width: f32,
+    /// The requested logical height of the window's client area.
+    /// May vary from the physical height due to different pixel density on different monitors.
     pub height: f32,
+    /// The position on the screen that the window will be centered at.
```

Comment:
I think this should explain a bit what happens when you pass it None

This is what the winit docs says:
```rust
    /// If this is `None`, some platform-specific position
    /// will be chosen.
    ///
    /// The default is `None`.
    ///
    /// ## Platform-specific
    ///
    /// - **macOS**: The top left corner position of the window content, the window's "inner"
    /// position. The window title bar will be placed above it.
    /// The window will be positioned such that it fits on screen, maintaining
    /// set `inner_size` if any.
    /// If you need to precisely position the top left corner of the whole window you have to
    /// use [`Window::set_outer_position`] after creating the window.
    /// - **Windows**: The top left corner position of the window title bar, the window's "outer"
    /// position.
    /// There may be a small gap between this position and the window due to the specifics of the
    /// Window Manager.
    /// - **X11**: The top left corner of the window, the window's "outer" position.
    /// - **Others**: Ignored.
    ///
```

Maybe just mentioning that the default position is controlled by the platform when None is used

Reply:
Mentioned it!